utils/db.py

```diff
@@ -0,0 +1,12 @@
+from django.db.models import DecimalField, Subquery
+
+
+class SQSum(Subquery):
```

Comment:
SQSum -> SQLSum?

Reply:
Comes from **S**_ub_**Q**_uery_**Sum** :)

Reply:
Ok, then I would just name it `SubquerySum`: it's not that long, and makes it more obvious what this does.